Projects/RevenueCat/Project.swift

```diff
@@ -0,0 +1,332 @@
+                "../../Tests/UnitTests/Misc/**/TestCase.swift",
+                "../../Tests/UnitTests/Misc/**/CustomerInfo+TestExtensions.swift",
+                "../../Tests/UnitTests/Misc/**/XCTestCase+Extensions.swift",
+                "../../Tests/UnitTests/Mocks/**/*.swift"
+            ],
+            resources: [
+                "../../Tests/StoreKitUnitTests/UnitTestsConfiguration.storekit"
+            ],
+            dependencies: [
+                .target(name: "RevenueCat"),
+                .target(name: "UnitTestsHostApp"),
+                .nimble,
+                .ohHTTPStubsSwift,
+                .snapshotTesting
+            ],
+            additionalFiles: [
+                "../../Tests/StoreKitUnitTests/UnitTestsConfiguration.storekit"
+            ]
+        ),
+
+        // MARK: – BackendIntegrationTests Host App
+        .target(
+            name: "BackendIntegrationTestsHostApp",
+            destinations: allDestinations,
+            product: .app,
+            bundleId: "com.revenuecat.StoreKitTestApp",
+            deploymentTargets: .multiplatform(
+                iOS: "16.0",
+                macOS: "11.0",
+                watchOS: "7.0",
+                tvOS: "14.0"
+            ),
+            infoPlist: .file(path: "../../Tests/UnitTestsHostApp/Info.plist"),
+            sources: [
+                "../../Tests/BackendIntegrationTestApp/**/*.swift"
+            ],
+            dependencies: [
+             .storeKit
+            ]
+        ),
+
+        .target(
+            name: "BackendCustomEntitlementsIntegrationTests",
+            destinations: allDestinations,
+            product: .unitTests,
+            bundleId: "com.revenuecat.BackendIntBackendCustomEntitlementsIntegrationTestsegrationTests",
+            deploymentTargets: .multiplatform(
+                iOS: "16.0",
+                macOS: "11.0",
+                watchOS: "7.0",
+                tvOS: "14.0"
+            ),
+            infoPlist: .default,
+            sources: [
+                "../../Tests/BackendIntegrationTests/CustomEntitlementsComputationIntegrationTests.swift",
+                "../../Tests/UnitTests/Misc/**/TestCase.swift",
+                "../../Tests/BackendIntegrationTests/BaseBackendIntegrationTests.swift",
+                "../../Tests/BackendIntegrationTests/BaseStoreKitIntegrationTests.swift"
+            ],
+            dependencies: [
+                .target(name: "RevenueCat"),
+                .target(name: "BackendIntegrationTestsHostApp"),
+                .nimble,
+                .ohHTTPStubsSwift,
+                .snapshotTesting,
+                .storeKitTests
+            ],
+        ),
+
+        .target(
+            name: "BackendIntegrationTests",
+            destinations: allDestinations,
+            product: .unitTests,
+            bundleId: "com.revenuecat.BackendIntegrationTests",
+            deploymentTargets: .multiplatform(
+                iOS: "16.0",
```

Comment:
same here I think we only need iOS

Reply:
do the others hurt?

Reply:
I think we can start adding those one by one. Let's try to keep everything as it is right now 🙏